README.md

```diff
@@ -996,6 +996,62 @@ func main() {
 }
 ```
 
+##### Perl using each
+
+```perl
+my %hash = ( key_1 => 'foo', key_2 => 'bar', );
+for my $key, $value ( each %hash ) {
+    printf( "key: %s value: %s\n", $key, $value );
+}
+
+```
+
+##### Go using only primary and secondary return values
+
+```go
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	myMap := map[string]string{"key1": "foo", "key2": "bar"}
+
+	for _, v := range myMap {
+		fmt.Printf("key: %s value: %s\n", k, v)
+	}
+}
+```
```

Comment:
For new Golang examples, I've been trying to add links to the Go playground as that makes it a bit easier to fiddle with examples and also just to make sure the examples are actually runnable. Do you think we could to that here?

Reply:
Yes, I am not so familiar with the playground integration, but I can generate some links.

Reply:
Links added